.github/workflows/releases.yaml

```diff
@@ -0,0 +1,46 @@
+name: Releases
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  PUSH: 'true'
+  DIRECT: 'true'
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  releases:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Log into registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
```

Comment:
qq: does GH provide a native directive for login? Not sure this plain `run` directive may leak the token? like display in the log publicly accessible?

Reply:
Updated